content/daytrip/eu/gb/adventure-valley.md

```diff
@@ -3,7 +3,7 @@ slug: "daytrip/eu/gb/adventure-valley"
 date: '2001-01-30T04:37:00'
 lat: '54.80711'
 lng: '-1.53963'
-location: Brasside, Framwellgate Moor, County Durham, North East, England, DH1 5SG, United Kingdom
+location: Brasside, Framwellgate Moor, County Durham, North East, DH1 5SG, United Kingdom
```

Comment:
I'm not going to comment on all of the ones like this, as there's a lot of them in this patch, but this is still too detailed. I'd lose "Framwellgate Moor" and "North East" (and probably retain "England", as we specify Scotland, Wales and NI elsewhere).